spec/message_repository_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'message_repository'
+require 'message'
+
+def reset_messages_table
+    seed_sql = File.read('spec/seeds/seeds_chitter_challenge_test.sql')
+    connection = PG.connect({ host: '127.0.0.1', dbname: 'chitter_challenge_test' })
+    connection.exec(seed_sql)
+end
+
+RSpec.describe MessageRepository do
+
+    before(:each) do 
+        reset_messages_table
+    end
+
+    context 'testing the MessageRepository class' do
```

Comment:
A minor note here and something I've only come to understand (or think I understand) more recently .. the context here should be the context in which the user is accessing the program, eg. 'when the user is logged in'. So that our test names describe the behaviour we want from our program, under certain conditions / contexts